todo_app/app.py

```diff
@@ -8,8 +15,40 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    print("index()")
+    items = sorted(get_items(), key=lambda item: item["status"], reverse=True)
+    for i in get_items():
+        print(i)
+    return render_template('index.html', items=items) 
 
+@app.route('/items', methods=['POST'])
+def add_new_item():
+    print("add_new_item")
+    title = request.form['title']
+    print(title)
+    add_item(title)
+    # HELP: how to redirect back to index
+    return index()
+
+@app.route('/mark_as_done/<itemId>') 
```

Comment:
It would be nice for this and the delete endpoint to be POST requests too, like the add_new_item method.

You'd need to use a `<form>` and `<button>` instead of an `<a>` tag in the HTML too.

Reply:
i'm not sure how to this inline... can you show me?
How to pass the id back with the form submit?

<!--<a class="card-link" href="{{ url_for('mark_as_done', itemId=item.id) }}">Done</a>   -->
                
                <form action="/mark_as_done/<itemId>" method="POST" class="form-group">
                    <button type="submit" class="btn btn-primary">Done</button>
                </form>

Reply:
Okay I've got it, thanks!

<form action="/mark_as_done/<{{item.id}}>" method="POST" class="form-group">
                    <button type="submit" class="btn btn-primary">Mark as Done</button>
                </form>